ios/Exponent/Kernel/ReactAppManager/EXReactAppManager.mm

```diff
@@ -626,6 +627,66 @@ - (void)toggleElementInspector
   }
 }
 
+- (void)toggleDevMenu
+{
+  if ([EXEnvironment sharedEnvironment].isDetached) {
+    [[EXKernel sharedInstance].visibleApp.appManager showDevMenu];
+  } else {
+    [[EXKernel sharedInstance] switchTasks];
+  }
+}
+
+- (void)setupWebSocketControls
+{
+  if ([self enablesDeveloperTools]) {
```

Comment:
let's wrap this conditional in `#if DEBUG || RCT_DEV`, like the other place in this file where we use `RCTPackagerConnection` -- related: https://github.com/expo/expo/commit/5026ff91ec71cd59cdc402117b1c47773252a582